guides/common/modules/proc_preparing-a-host-for-external-databases.adoc

```diff
@@ -24,8 +24,14 @@ ifndef::orcharhino[]
 [options="nowrap" subs="+quotes,attributes"]
 ----
 # subscription-manager repos --disable '*'
+ifdef::satellite[]
 # subscription-manager repos --enable={RepoRHEL7ServerSoftwareCollections} \
 --enable={RepoRHEL7Server} --enable={RepoRHEL7ServerSatelliteServerProductVersion}
+endif::[]
+ifndef::satellite[]
+# subscription-manager repos --enable={RepoRHEL7ServerSoftwareCollections} \
+--enable={RepoRHEL7Server}
```

Comment:
Since everything is disabled prior to this, won't you miss the Foreman repositories which you need for the DB extension?

Reply:
Ah ok.